gulpfile.js

```diff
@@ -1,73 +1,90 @@
-/* global console */
+'use strict'; // eslint-disable-line strict
 /* eslint no-console:0 */
+const spawn = require('child_process').spawn;
+const denodeify = require('denodeify')
 const gulp = require('gulp');
+const Server = require('karma').Server;
 const obt = require('origami-build-tools');
-const nodemon = require('gulp-nodemon');
 const path = require('path');
+const selenium = require('selenium-standalone');
+
+const installSelenium = denodeify(selenium.install.bind(selenium));
```

Comment:
This will install the latest version of Selenium and the default WebDrivers - Chrome & IE.